plexmuxy_gui/static/index.html

```
@@ -0,0 +1,158 @@
+
+      <div id="alert" class="alert hidden" role="alert"></div>
+
+      <section id="directory-section" class="panel">
+        <div class="panel-header">
+          <div>
+            <p class="eyebrow">Directory</p>
+            <h3>Input directory</h3>
+          </div>
+          <button id="choose-dir-btn" type="button">Choose folder</button>
+        </div>
+        <input id="input-dir" type="text" placeholder="No directory selected" autocomplete="off" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)